packages/stellar-axelar-std/src/interfaces/mod.rs

```diff
@@ -33,10 +33,14 @@ mod storage {
     // because the key serialization is variant order dependent.
 
     pub mod operator {
-        use soroban_sdk::contracttype;
+        use soroban_sdk::Address;
 
-        #[contracttype]
-        pub enum DataKey {
+        use crate::contractstorage;
```

Comment:
does it work if you import via axelar-std-derive?

Reply:
might be an issue with the import order. it's not available under `crate::` yet
https://github.com/axelarnetwork/axelar-cgp-stellar/blob/main/packages/stellar-axelar-std/src/lib.rs#L31